src/main/java/org/folio/services/instance/InstanceService.java

```diff
@@ -124,7 +125,12 @@ public Future<Response> updateInstance(String id, Instance newInstance) {
     return refuseLongNotes(newInstance)
       .compose(notUsed -> instanceRepository.getById(id))
       .compose(CommonValidators::refuseIfNotFound)
-      .compose(oldInstance -> refuseWhenHridChanged(oldInstance, newInstance))
+      .compose(oldInstance -> {
+        if (!newInstance.getSource().startsWith("CONSORTIA-")) {
```

Comment:
please consider defining `"CONSORTIA-"` as a constant

Reply:
This string is only used in one place. I don't think we need an additional constant for this.

Reply:
Please rename it to CONSORTIUM

Reply:
thanks, completed